lib/components/primitive-components/Group/Group.ts

```diff
@@ -112,7 +104,7 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
     }
   }
 
-  doInitialPcbTraceRender() {
+  async doInitialPcbTraceRender() {
```

Comment:
render functions can't be async

```suggestion
  doInitialPcbTraceRender() {
```

Reply:
Yeah, I knew we can't async the renderPhases. But this was the way to synchronise the two calls to the autorouter